### ISSUE TITLE
Error redirect uri with chrome using twitter login

### ISSUE THREAD

**Issue by Hadrien-DELAITRE** [MrSwitch/hello.js#171]
hi, sorry for my english ...

I try to use twitter login with hellojs.
1) in twitter api "Callback URL" is set to my default page login
2) i configured "https://auth-server.herokuapp.com/#signin" with my id and my secret and with the domain of my site

With IE and Firefox, no problem: there is a twitter popup. I can connect and the popup closes automatically. Then i retrieves the data correctly provided by twitter.

With chrome, no way... the popup never closes. 
When i do a "CTRL+F5" in the popup, at that time she closes and i can retrieve data provided by twitter.

Best regards,

**Comment by SinisaG** [comment]
Having the same issue. Problem occurs when I load the script dynamically. 

E.g.: 

```
            function async(u, c) {
                var d = document, t = 'script',
                        o = d.createElement(t),
                        s = d.getElementsByTagName(t)[0];
                o.src = '//' + u;
                if (c) { o.addEventListener('load', function (e) { c(null, e); }, false); }
                s.parentNode.insertBefore(o, s);
            }

         async(window.location.host  + '/bower_components/hello/dist/hello.all.min.js', start);
```

Any idea what could go wrong here? It happens only for twitter on Chrome (and Opera, can't try Safari), works in IE and Firefox. 

I use you auth proxy (https://auth-server.herokuapp.com/proxy). 

It works if I use a normal script tag, which is, as you put it, perplexing. :)

**Comment by vanwars** [comment]
I also just experienced this issue: I try to authenticate to Twitter (also using your auth proxy), and the popup hangs with no error message in Chrome. I was previously using require.js to load the hello.all.min.js file, but based on @SinisaG's suggestion, I  switched to using a static reference to the file in the head tag. It works now! Thank you for sharing your solution. Definitely perplexing (I tried to track down the bug for a couple of hours, but no luck).
